microchip/common/modules.nix

```diff
@@ -0,0 +1,11 @@
+{ pkgs, lib, ... }: {
+  nixpkgs.overlays = [
+    (import ./overlay.nix)
```

Comment:
The overlay seems unnecessary here. Each overlay is an expensive copy of the whole nixpkgs attribute set and overlays that just add a single package seem a bit overkill.

Reply:
https://zimbatm.com/notes/1000-instances-of-nixpkgs for more details.